adagrams/game.py

```diff
@@ -1,13 +1,139 @@
 from random import randint
 
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
 def draw_letters():
-    pass
+    pool_list = []
+
+    for letter, qty in LETTER_POOL.items():
+        for _ in range(qty):
+            pool_list.append(letter)  
+
+    hand = []
+    for _ in range(10):
+        idx = randint(0, len(pool_list) -1)
+        hand.append(pool_list.pop(idx))
+
+    return hand
+
 
 def uses_available_letters(word, letter_bank):
-    pass
+    counts = {}
+    for ch in letter_bank:
+        ch = ch.upper()
+        if ch in counts:
+            counts[ch] += 1
+        else:
+            counts[ch] = 1
```

Comment:
Great idea to create a frequency dictionary for your letter bank! This creates an easy structure for lookup and it acts as a copy of the letter bank that we can remove values from without affecting the original! 

Did you know you can create a frequency dictionary in one line using the .get() method? That would look something like the following:

```suggestion
    for ch in letter_bank:
        ch = ch.upper()
        counts[ch] = counts.get(ch,0) + 1
```
This sets the default of a key it doesn't find to 0 and then adds one. It then stores the new value in the dictionary! This is a really succinct and useful way to build frequency maps!